lib/hash-tree.js

```diff
@@ -33,7 +33,18 @@ function stringifyFileInfo(fileInfo) {
 
 module.exports = function hashTree(fullPath) {
   var strings = getFileInfos(fullPath).map(stringifyFileInfo).join();
-  return crypto.createHash('md5').update(Buffer.from(strings, 'utf8')).digest('hex');
+
+  var buf;
+  // Once we drop old new support, we can simplify this code to Buffer.from
+  if (typeof Buffer.from === 'function') {
+    buf = Buffer.from(strings);
+  } else {
+    // Old Node versions don't support Buffer.from, so use 'new' instead.
+    buf = new Buffer(strings, 'utf8');
+
+  }
+  var hash = crypto.createHash('md5').update(buf).digest('hex');
```

Comment:
Any reason not to return here directly?

Reply:
Not really, though sometimes for debugging it's nice to put the value into a variable first so you can check it in the debugger. Do you guys have a preference in those sorts of situations?

Reply:
Switched to just returning the expression value instead of assigning first.